pyproject.toml

```diff
@@ -21,13 +21,13 @@ gen3authz = "^1.0.4"
 hsclient = {git = "https://github.com/uc-cdis/hsclient", rev = "1.0.0"}
 indexclient = "^2.1.0"
 jsonschema = "^3.2"
+psycopg2-binary = "^2.7"
+gunicorn = "^22.0.0"
```

Comment:
If we are not blocked from upgrading beyond these versions, I'd like to get in the habit of specifying >= rather than the default ^. If things break, we'll see it in unit tests and either update so the newer lib works, or at that point we'll restrict the versioning and create a ticket to upgrade. 

As it is, we'll never automatically try to update to new major versions.

So, even though the other deps aren't like this _yet_, we could replace all ^ with >= at least with the new deps. 

If you're feeling adventurous, we could try everything else too. With the exception of sqlalchemy, which I know won't work b/c 2.0 requires updates